clickhouse-client/src/main/java/com/clickhouse/client/config/ClickHouseDefaultSslContextProvider.java

```diff
@@ -150,6 +150,20 @@ protected SSLContext getJavaSslContext(ClickHouseConfig config) throws SSLExcept
                     factory.init(getKeyStore(sslRootCert, null));
                     tms = factory.getTrustManagers();
                 }
+
+                if (truststorePath != null && !truststorePath.isEmpty()) {
```

Comment:
Can we take trustStore as priority and only use sslRootCert when it's not specified?